paper_trail.gemspec

```diff
@@ -28,7 +28,7 @@ has been destroyed.
   s.required_ruby_version = ">= 2.3.0"
 
   # Rails does not follow semver, makes breaking changes in minor versions.
-  s.add_dependency "activerecord", [">= 4.2", "< 5.2"]
+  s.add_dependency "activerecord", [">= 4.2", "< 6"]
```

Comment:
Because rails makes breaking changes in minor versions, I'd prefer `< 5.3` here.

Reply:
There is no 5.3. After 5.2 it will be 6.0.

Reply:
> There is no 5.3. After 5.2 it will be 6.0.

Of course, if you say this, I believe you :) You are the person who would know. But, it doesn't change my policy for this version constraint. What advantage does PaperTrail gain by specifiying `< 6` rather than `< 5.3`? Maybe I am missing something.

Reply:
I think the only advantage is make sure that we are using a version constraint that actually exists, but it doesn't matter if we use 6.0, so I'll change.